doc/manuals/admin/install.md

```diff
@@ -12,20 +12,20 @@
 
 ## Introduction
 
-The recommended procedure is to install using RPM packages in CentOS 6.x. If you are interested in
+The recommended procedure is to install using RPM packages in CentOS 7.x. If you are interested in
 building from sources, check [this document](build_source.md).
 
 ## Requirements
 
 * System resources: see [these recommendations](diagnosis.md##resource-availability)
-* Operating system: CentOS/RedHat. The reference operating system is CentOS 6.3
-  but it should work also in any later CentOS/RedHat 6.x version.
+* Operating system: CentOS/RedHat. The reference operating system is CentOS 7.4.1708
+  but it should work also in any later CentOS/RedHat 7.x version.
```

Comment:
and possible in earlier too ?
Or will the changes in this PR give problems in CentOS 6 ?

Reply:
It will not give problems with CentOS 6, but I think is cleaner to refer to the current version (and from no on current = CentOS 7) and the future.

NTC

Reply:
Sure, but ...

I think we should mention that the broker still works in CentOS 6. After all, most of our client base probably use that distro. They might get worried ...

Also, we will no longer make sure the broker works in CentOS 6, right?
If so, that we should mention as well

Reply:
NTC